create.ps1

```diff
@@ -67,80 +50,115 @@ function Get-ADPAccessToken {
 }
 
 
-function Resolve-HTTPError {
+function Resolve-ADPWorkforceError {
```

Comment:
I would suggest to __always__ copy the error handling function from the V1 connector since this is proven. The default function in the template always requires modification. The default function 'might' work but currently doesn't handle the friendly message correctly as this still contains a raw JSON.

So, in this case you can either keep the current function but it does require some testing to make sure all the properties within the `$httpErrorObj` contain the right information. 

Or, just copy/paste the error handling `Resolve-Error` function from the V1.

Reply:
solved